fluent-dom/src/overlay.js

```diff
@@ -71,8 +71,14 @@ export default function overlayElement(element, translation) {
     return;
   }
 
+  const whitelistedAttrs = element.hasAttribute('data-l10n-attrs') ?
+    element.getAttribute('data-l10n-attrs')
+      .split(',').map(attr => attr.trim()) :
+    null;
```

Comment:
I'd prefer the following indentation:

    declaration = predicate
      ? consequent
      : alternate;

Or, since the alternate isn't very interesting (`null`) you could also use a `var` inside of an `if` statement.